pkg/models/gacha.go

```diff
@@ -101,13 +118,12 @@ func GetUserByToken(token string) []User {
 	return users
 }
 
-func GetAllUsers(token string) []User {
-
+func GetAllUsers(token string) ([]User, error) {
```

Comment:
I thought it didn't need this function?

Reply:
Yes that function is not required anymore.I will comment that part in the new PR.